docs/src/installation_instructions.md

```diff
@@ -1,25 +1,76 @@
-# Installation instructions
+# Getting started
 
-You can install the latest version of Oceananigans using the built-in package manager (accessed by pressing `]` in the
-Julia command prompt) to add the package and instantiate/build all dependencies
+This section covers the basics of installing Julia and Oceananigans, and then describes best practices when running
+Oceananigans. If you're new to Julia, we also recommend reading [Julia's "getting started" page](https://julialang.org/learning/getting-started/).
+
+
+## Installation instructions
+
+The hardest part of installing Oceananigans is actually installing Julia (and even that should be pretty easy). 
+If you want to use Oceananigans on a laptop you can generally install Julia easily via the package manager of whatever
+operational system you use (some quick googling should tell you how). If you're running on a cluster, you should check
+if it already has Julia installed (most systems these days do). If that's the case, then the hardest part is already done!
+
+If those options above are not possible, one alternative is to simply download the Julia pre-compiled binaries from the
+[Julia website](https://julialang.org/downloads/), where you can simply run the Julia executable that's contained inside
+(this should work out of the box!). Another option is to compile Julia manually from source code. This is definitely the hardest
+option, and you can find instructions on how to do that [here](https://github.com/JuliaLang/julia#building-julia).
```

Comment:
This should be more concise. It's hard to skim these paragraphs and glean useful information. I don't think it's useful to say "installing Julia should be pretty easy". Users will determine for themselves whether its easy. We should have just 1-2 sentences about installing julia with relevant links I think.

Reply:
Done